### PR TITLE
Update sweden-scilifelab.md

### PR DESCRIPTION
**Pull request by FranBonath**

### CODE REVIEW

**Comment by FranBonath** [comment]
@nf-core-bot fix linting

File: markdown/events/2023/hackathon-march-2023/sweden-scilifelab.md [hunk 1]
```diff
@@ -16,4 +16,81 @@ Local event to be held at SciLifeLab, Stockholm.
...
+                <td data-timestamp="1679925600"  data-timeformat="HH:mm z">16:00</td>
+                <td rowspan="1">Hub Check In</td>
+                <td rowspan="1">Hub Check In</td>
+                <td rowspan="1">Hackathon Summary</td>
+            </tr>
+            <tr>
+                <td data-timestamp="1679927400" data-timeformat="HH:mm z">16:30</td>
+                <td rowspan="2">Hack!</td>
+                <td rowspan="2">Hack!</td>
+                <td rowspan="4">End of Hackathon social</td>
```
Comment (mashehu) [inline markdown/events/2023/hackathon-march-2023/sweden-scilifelab.md:79]:
```suggestion
                <td rowspan="4">Hackathon social in Delta 2</td>
```
Or where would the social be?

Comment (FranBonath) [reply markdown/events/2023/hackathon-march-2023/sweden-scilifelab.md:79]: But more likely in Alfa3 or Gamma2, since that's where the drinks are.

Comment (mashehu) [reply markdown/events/2023/hackathon-march-2023/sweden-scilifelab.md:79]: okay, then it is fine. but maybe a short description about what the social event will be?

Comment (mashehu) [reply markdown/events/2023/hackathon-march-2023/sweden-scilifelab.md:79]: yeah, I meant gamma2 🙃

**Comment by FranBonath** [comment]
@mashehu is this ok for now to be published?

**Review by mashehu — APPROVED** [review]

**Pull request merged**